packages/client/src/v2-events/features/events/actions/correct/request/Summary/Summary.tsx

```diff
@@ -96,17 +90,23 @@ function setEmptyValuesForFields(fields: FieldConfig[]) {
   }, {})
 }
 
+const SectionTitle = styled(Text)`
+  margin-bottom: 20px;
+`
+
+const CorrectionInformationSectionTitle = styled(SectionTitle)`
+  margin-top: 20px;
+`
```

Comment:
Not sure if it helps here but we have a wrapper component that applies margin to it's children

https://develop.opencrvs.pages.dev/?path=/docs/layout-stack--docs

Reply:
Avoiding the hardcoded pixels helps us move towards https://github.com/opencrvs/opencrvs-core/issues/9017 in the near future. Eezi has already opened a PR